webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -201,6 +213,7 @@ class SubscriptionsPage extends Component {
     });
 
     const onDeleteSubscriptions = (selectedRows) => {
+      this.startManifestTask();
       this.props.deleteSubscriptions(selectedRows);
```

Comment:
I don't see anywhere `selectedRows` is updated after deleting.  Shouldn't `selectedRows` be cleared out after deleting?

Reply:
Yes you are right. didn't push the right code. I have updated it now.